pkg/cli/admin/toppvc/persistentvolumeclaims.go

```diff
@@ -159,10 +159,10 @@ func (o *options) Run(ctx context.Context, args []string) error {
 
 	if len(promOutput.Data.Result) == 0 {
 		if o.namespace == "" {
-			return fmt.Errorf("no persistentvolumeclaims found.")
+			return fmt.Errorf("no persistentvolumeclaims found or mounted.")
```

Comment:
```suggestion
			return fmt.Errorf("no persistentvolumeclaims found in use.")
```
Maybe better, `mount` generally for pods related info. Anyway let's also wait for other guys thoughts.

Reply:
I agree on this one with @Phaow in both scenarios.

Reply:
It makes sense to avoid using mount or attach word.
Maybe the following is a bit grammatically :) 

> no PersistentVolumeClaims **are** found in use.